html/index.html

```diff
@@ -1,17 +1,337 @@
+						<i class="fa fa-google-plus-official" aria-hidden="true"></i>
+					</a>
+				</div>
+			</div>
+
+
+			<div class="team-one">
+				<img src="images/2.png" alt="">
+				<h4>Dr. Shipon Kumar</h4>
+				<h5>Physical Medicine, MBBS</h5>
+				<p>Pharetra ultricies aenean, sit metus integer arcu turpis dndimentum pellentesque world</p>
+				<div class="social-icons-wrap">
+					<a href="https://www.instagram.com/" class="facebook" target="_blank">
+						<i class="fa fa-facebook" aria-hidden="true"></i>
+					</a>
+
+					<a href="twitter" class="twitter">
+						<i class="fa fa-twitter" aria-hidden="true"></i>
+					</a>
+
+					<a href="google-plus" class="google-plus">
+						<i class="fa fa-google-plus-official" aria-hidden="true"></i>
+					</a>
+				</div>
+
+			</div>
+			<div class="team-one">
+				<img src="images/3.png" alt="">
+				<h4>Dr. Marina Yigo</h4>
+				<h5>Physical Medicine, MBBS</h5>
+				<p>Pharetra ultricies aenean, sit metus integer arcu turpis dndimentum pellentesque world</p>
+				<div class="social-icons-wrap">
+					<a href="https://www.instagram.com/" class="facebook" target="_blank">
+						<i class="fa fa-facebook" aria-hidden="true"></i>
+					</a>
+
+					<a href="twitter" class="twitter">
+						<i class="fa fa-twitter" aria-hidden="true"></i>
+					</a>
+
+					<a href="google-plus" class="google-plus">
+						<i class="fa fa-google-plus-official" aria-hidden="true"></i>
+					</a>
+				</div>
+			</div>
+		</div>
+	</div>
+</div>
+
+<div class="signup-section">
+	<h5>Signup for Our Newslatter</h5>
+
+	<div class="newsletter-area">
+		<form action="#" class="wow fadeInUp" data-wow-delay="0.4s">
+			<input type="email" placeholder="Enter Your Email Address" required="">
+			<button type="submit" class="primary-light-bg"><i class="fa fa-paper-plane"></i></button>
+		</form>
+	</div>
+</div>
+
+<div class="latest-update-section">
+	<div class="wrapper">
+		<div class="latest-update-section-one">
+			<h2>Latest Update</h2>
+			<p>Condimentum rutrum placerat egestas condimentum mi eros. Eleifend cras quirntum</br>Feugiat elit
+				placerat. Diam tempor malesuada</p>
+		</div>
+		<div class="latest-update-section-two">
+			<div class="admin-section">
+				<img src="images/1 (1).jpg" alt="admin">
+				<div class="date">01 JULY 2019</div>
+				<div class="admin-content">
+					<div class="aurthor-info">
+						<div class="admin-wrap">
+							by
+							<a href>Admin</a>
+						</div>
+						<div class="profile-info">
+                            <span>
+                                <a href>doctor</a>
+
+                                <a href>Clinic</a>
+                            </span>
+						</div>
+					</div>
+                    <h2>Remember Outweigh Do He Desirous No Cheerful</h2>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore</p>
+                    <a href="" class="bttn-mid btn-emt">Read More</a>
```

Comment:
`href` value is missing.
Please put the value of `href` e.g.
` <a href="http://google.com" class="bttn-mid btn-emt">Read More</a>`